frontend/javascripts/admin/organization/organization_cards.tsx

```diff
@@ -0,0 +1,338 @@
+            : UpgradePricingPlanModal.upgradeUserQuota
+        }
+      >
+        <PlusCircleOutlined /> Upgrade
+      </span>,
+    ];
+    upgradeStorageAction = [
+      <span
+        key="upgradeStorageAction"
+        onClick={
+          organization.pricingPlan === PricingPlanEnum.Free
+            ? () => UpgradePricingPlanModal.upgradePricingPlan(organization)
+            : UpgradePricingPlanModal.upgradeStorageQuota
+        }
+      >
+        <PlusCircleOutlined /> Upgrade
+      </span>,
+    ];
+    upgradePlanAction = [
+      [
+        <a
+          href="https://webknossos.org/pricing"
+          target="_blank"
+          rel="noreferrer"
+          key="comparePlanAction"
+        >
+          <SafetyOutlined /> Compare Plans
+        </a>,
+      ],
+    ];
+  }
+
+  return (
+    <>
+      {hasPlanExceeded ? (
+        <PlanExceededAlert organization={organization} />
+      ) : (
+        <PlanAboutToExceedWarning
+          organization={organization}
+          usedStorageSpace={usedStorageSpace.usedStorageSpace}
+        />
+      )}
+      <Row gutter={24} justify="space-between" align="stretch" style={{ marginBottom: 20 }}>
+        <Col>
+          <Card actions={upgradeUsersAction}>
+            <Row style={{ padding: 20 }}>
+              <Progress
+                type="dashboard"
+                percent={usedUsersPercentage}
+                format={() => `${activeUsersCount}/${maxUsersCountLabel}`}
+                strokeColor={hasExceededUserLimit ? redStrokeColor : greenStrokeColor}
+                status={hasExceededUserLimit ? "exception" : "active"}
+              />
+            </Row>
+            <Row justify="center">Users</Row>
+          </Card>
+        </Col>
+        <Col>
+          <Card actions={upgradeStorageAction}>
+            <Row style={{ padding: 20 }}>
+              <Progress
+                type="dashboard"
+                percent={usedStoragePercentage}
+                format={() => storageLabel}
+                strokeColor={hasExceededStorageLimit ? redStrokeColor : greenStrokeColor}
+                status={hasExceededStorageLimit ? "exception" : "active"}
+              />
+            </Row>
+            <Row justify="center">Storage</Row>
+          </Card>
+        </Col>
+        <Col>
+          <Card actions={upgradePlanAction}>
+            <Row justify="center" align="middle" style={{ minHeight: 160, padding: "25px 35px" }}>
+              <h3>{organization.pricingPlan}</h3>
+            </Row>
+            <Row justify="center">Current Plan</Row>
+          </Card>
+        </Col>
+      </Row>
+    </>
+  );
+}
+
+function PlanExceededAlert({ organization }: { organization: APIOrganization }) {
+  const hasPlanExpired = Date.now() > organization.paidUntil;
+
+  const message = hasPlanExpired
+    ? "Your webKnossos plan has expired. Renew your plan now to avoid being downgraded and lose access to features and prevent users from being blocked."
```

Comment:
```suggestion
    ? "Your webKnossos plan has expired. Renew your plan now to avoid being downgraded, users being blocked, and losing access to features."
```